### PR TITLE
Update client-failover.md

### PR DESCRIPTION
**Pull request by zryska**
Including a note that a separate Adapter instance should be setup to perform history recovery when the Adapter has client failover enabled.

### CODE REVIEW

**Review by AnnonsonTechWriter — DISMISSED** [review]
Is "On Demand history recovery" the same as on-demand history recovery? Should standardize in one or the other.

**Comment by zryska** [comment]
> Is "On Demand history recovery" the same as on-demand history recovery? Should standardize in one or the other.

Thanks Bob! I noticed that we use On Demand history recovery in the feature description so I standardized on that.

**Comment by AnnonsonTechWriter** [comment]
Excellent. Thank you.

**Comment by Xiaoli-OSI** [comment]
Thanks for notifying me with the change. I have informed Jane this change is coming to V1.7 user guide.

Regards,
Xiaoli

**Comment by Xiaoli-OSI** [comment]
Xiaoli Tang would like to recall the message, "[osisoft/PI-Adapter] Update client-failover.md (PR #127)".
__________________________________________________________________

AVEVA Group plc is registered in England at High Cross, Madingley Road, Cambridge, England CB3 0HB. Number 2937296.

**Review by Xiaoli-OSI — APPROVED** [review]

File: shared-content/configuration/client-failover.md [hunk 1]
```diff
@@ -88,6 +88,8 @@ The following is an example of a complete client failover configuration.
 
 **Note:** You can only register the adapter to a single failover endpoint by providing one client failover configuration. Additional configuration entries will be rejected.
 
+**Note:** When On Demand history recovery is required a new AVEVA Adapter instance should be configured to perform the operation. It isn't recommended to use On Demand history recovery on an Adapter instance that participates in a failover pair.
```
Comment (aveva-jhwang) [inline shared-content/configuration/client-failover.md:91]: Could you add a comma after "required" in the first sentence? Also, I feel like the second statement should be firmer than a recommendation. If the customer configures it in this way, they may see adapter role changes that are not expected nor desirable. Bob, do you have any ideas on how we could rephrase it?

**Pull request merged**